lib/x11test.pm

```diff
@@ -521,7 +521,7 @@ sub start_clean_firefox {
 
     x11_start_program('xterm');
     # Clean and Start Firefox
-    type_string "killall -9 firefox;rm -rf .moz* .config/iced* .cache/iced* .local/share/gnome-shell/extensions/*; firefox suse.com >firefox.log 2>&1 &\n";
+    type_string "killall -9 firefox;rm -rf .moz* .config/iced* .cache/iced* .local/share/gnome-shell/extensions/*; firefox html5test.opensuse.org >firefox.log 2>&1 &\n";
```

Comment:
Even more lightweight would be http://static.opensuse.org/ when we do not want to test the rendering of the html5 page itself. How about using html5test in a single test module and static in all others?

Reply:
it has to trigger tracking pop-up

Reply:
but http://static.opensuse.org could be used as homepage, I will try to use it